cccs-build/superset/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM uchimera.azurecr.io/cccs/superset-base:cccs-1.4_20220426124825_b3385
+FROM uchimera.azurecr.io/cccs/superset-base:feature_CLDN-1397_20220510175708_b3440
```

```suggestion
FROM uchimera.azurecr.io/cccs/superset-base:cccs-1.4_20220426124825_b3385
```

Since you temporarily modified this for deploying to scylladev, we should change it back to what it was.